packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -960,7 +960,7 @@ describeSpec('Limbo Documents:', [], () => {
           // docAs replaced them with docBs. If Watch has to re-run the
           // underlying query when this client re-listens, Watch won't be able
           // to tell that docAs were deleted and will only send us existing
-          // documents that changed since the resume token. This will cause it
+          // documents that matches the query since the resume token. This will cause it
```

Comment:
🤔  I think watch send all docs since the resume token. This is the test cases that have communicated with Xijiao:https://docs.google.com/document/d/12M6KH3y6eZDTOuQX_tfRKgf5v-V_Nh-pn27D9OwqfGE/edit?resourcekey=0-32ewb_qRIuitZKe-Nf0PpA. DocA is barely touched, but it is always send in the list.